src/index.js

```diff
@@ -14,15 +14,14 @@ let initModule = false;
  * @param  {string} [config.LOGS_TOKEN] Token used by internal logger to send logs to
  * @return {Object}        The configured providers
  */
-module.exports = function(config) {
-  //if no param is provided, we want to return the cached modules
+module.exports = (config) => {
+  // if no param is provided, we want to return the cached modules
   if (!config) {
```

Comment:
If it complicates things then really no need to od anything since it is an edge case and we usually control the input param (os less chance something programatically sets it to {})

@ky23 it's the jf(!config) part, it is not falsy so it will not enter that part of code.